src/actions/index.js

```diff
@@ -20,15 +20,22 @@ const createActions = ({name, pluralName, url: defaultUrl, actions = {}, credent
     const actionOpts = {...actions[actionKey], credentials};
     const reducerOpts = pick(actionOpts, 'assignResponse');
     const type = getActionType({name, action, actionKey});
-    const url = action.url || defaultUrl;
-    const urlParams = parseUrlParams(url);
+    const actionUrl = action.url || defaultUrl;
     // Compute actual function name
     const actionName = getActionName({name, pluralName, actionKey, actionOpts});
     // Actual action function
-    const actionFunc = (context, contextOpts = {}) => (dispatch) => {
+    const actionFunc = (context, contextOpts = {}) => (dispatch, getState) => {
       // First dispatch a pending action
       dispatch({type, status: 'pending', context});
+      let url = actionUrl;
+      if (typeof actionUrl === 'function') { url = actionUrl(getState); }
```

Comment:
@mgcrea If this returns a Promise, should it wait for it to resolve and then use this value ?